src/config.rs

```diff
@@ -83,6 +86,15 @@ pub struct RusticConfig {
     pub webdav: Option<Value>,
 }
 
+impl Display for RusticConfig {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        let config =
+            toml::to_string_pretty(self).unwrap_or_else(|_| "Error serializing config".to_string());
```

Comment:
```suggestion
            toml::to_string_pretty(self).unwrap_or_else(|_| "<Error serializing config>".to_string());
```

This might be an error case we **never** encounter.
Anyway I propose some way to clarify that the error case in the display is some <placeholder>.